django/tasks/backends.py

```diff
@@ -0,0 +1,119 @@
+from . import constants
+from .base import get_backend
+from .exceptions import ResultUnavailable
+
+
+class TaskResult(object):
+    """
+    Wrapper class representing a status of a task. Provides methods to retrieve
+    the task's status and result value/error.
+    """
+    def __init__(self, backend, task_id):
+        if not isinstance(backend, BaseBackend):
+            backend = get_backend(backend)
+        self._backend = backend
+        self.task_id = task_id
+
+    def __repr__(self):
+        return '<TaskResult %s - %r>' % (self.alias, self.task_id)
+
+    @property
+    def alias(self):
+        return self._backend.alias
+
+    def get_status(self, **kwargs):
```

Comment:
Why does `get_status()` need kwargs, ere and everywhere else? Isn't the taskid enough?